lib/api/route/saved_snippets.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:json_annotation/json_annotation.dart';
+
+import '../core.dart';
+
+part 'saved_snippets.g.dart';
+
+/// https://zulip.com/api/create-saved-snippet
+Future<CreateSavedSnippetResult> createSavedSnippet(ApiConnection connection, {
+  required String title,
+  required String content,
+}) {
+  return connection.post('createSavedSnippet', CreateSavedSnippetResult.fromJson, 'saved_snippets', {
```

Comment:
```suggestion
}) {
  assert(connection.zulipFeatureLevel! >= 297); // TODO(server-10)
  return connection.post('createSavedSnippet', CreateSavedSnippetResult.fromJson, 'saved_snippets', {
```

That doubles as a reminder, when writing code that calls this endpoint, of what feature level to check for before offering the feature in the UI.